vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors_moe.py

```diff
@@ -131,6 +131,27 @@ def __init__(
 
         self.rocm_aiter_moe_enabled = is_rocm_aiter_moe_enabled()
 
+
```

Comment:
duplicate code with above compressed_tensors.py. is it possible to move this to a common place?

Reply:
Updated.

Reply:
what's the meaning of "rocm_aiter" :)